R/pkg/inst/tests/test_sparkSQL.R

```diff
@@ -108,6 +108,14 @@ test_that("create DataFrame from RDD", {
   expect_equal(count(df), 10)
   expect_equal(columns(df), c("a", "b"))
   expect_equal(dtypes(df), list(c("a", "int"), c("b", "string")))
+
+  localDF <- data.frame(name=c("John", "Smith", "Sarah"), age=c(19, 23, 18), height=c(164.10, 181.4, 173.7))
+  schema <- structType(structField("name", "string"), structField("age", "integer"), structField("height", "float"))
+  df <- createDataFrame(sqlContext, localDF, schema)
```

Comment:
For user specified schema for createDataFrame, my point is we may not support some DataTypes like byte, long, float, which is not natural to R users. Or alternatively, from the view point of API parity with Scala, we support these types but internally convert to R natural types, like:
   byte ->  integer
  long -> double
  float -> double
and print some warning message about the conversion.